bootstrap/src/main/java/io/jenkins/jenkinsfile/runner/bootstrap/Util.java

```diff
@@ -0,0 +1,49 @@
+package io.jenkins.jenkinsfile.runner.bootstrap;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Enumeration;
+import java.util.jar.JarEntry;
+import java.util.jar.JarFile;
+
+
+public class Util {
+
+    public static File explodeWar(String jarPath) throws IOException {
+        JarFile jarfile = new java.util.jar.JarFile(new java.io.File(jarPath));
+        Enumeration<java.util.jar.JarEntry> enu= jarfile.entries();
+
+        //Creating working directory
```

Comment:
Since you unpack the WAR every time anyway, it would be better to use a temporary directory. Also, using a generic `work` directory might be a problem for other cases, because "work" in Jenkinsfile Runner is likely to represent a workspace instead

Reply:
Yeah created a temporary directory which in linux systems would create something like this
`/tmp/java-5122075410590166027`.  So I created a temp dir in the current working directory.